.github/CONTRIBUTING.md

```diff
@@ -49,26 +49,30 @@ Example:
 
 ## Feature requests
 
-Feature requests are welcome. But take a moment to find out whether your idea fits with the scope and aims of the project. It's up to *you* to make a strong case to convince the project's developers of the merits of this feature. Please provide as much detail and context as possible.
+Feature requests are welcome but take a moment to find out whether your idea fits with the scope and aims of the project. It's up to *you* to make a strong case to convince the project's developers of the merits of this feature. Please provide as much detail and context as possible.
 
 ## Pull requests
 
 Good pull requests—patches, improvements, new features—are a fantastic help. They should remain focused in scope and avoid containing unrelated commits.
 
 **Please ask first** before embarking on any significant pull request (e.g. implementing features, refactoring code, porting to a different language), otherwise you risk spending a lot of time working on something that the project's developers might not want to merge into the project.
 
-Adhering to the following process is the best way to get your work included in the project:
+### Updating Primer modules
 
-1. Fork and clone the repository
-2. Configure and install the dependencies: `bower install`
-3. Create a new branch: `git checkout -b my-branch-name`
-4. Make your change, add tests, and make sure the tests still pass
-5. Push to your fork and [submit a pull request][pr]
-6. Pat your self on the back and wait for your pull request to be reviewed and merged.
+Anyone can open a pull request on Primer CSS. You do not need to work at GitHub or be a member of the org to open a pull request. 
+
+1. Fork and clone the primer/primer-css repository.
```

Comment:
Sure, though I might link it. It seems obvious to us but with the new setup people might get muddled about where pr's should happen.